### ISSUE TITLE
Update docs for Input Polling

### ISSUE THREAD

**Issue by Cleptomania** [pythonarcade/arcade#1052]
#1038 added the ability to do input polling to the Window class. Need to add examples to the documentation demonstrating this, and maybe cover the difference between event based input and polling during the input section of the platformer tutorial.

**Comment by Cleptomania** [comment]
I don't think this is necessary to get done for 3.0. Would be nice to have at some point but it's a somewhat niche feature mostly useful in quick testing and prototyping.

**Comment by einarf** [comment]
Agreed. These are things we can iron after release.

**Comment by einarf** [comment]
Status on this?
